### DISCUSSION TITLE
Setting the caption position in a table

### DISCUSSION THREAD

**Discussion by ajcanepa** [quarto-dev/quarto-cli#10210] [Q&A] [answered]
### Description

Setting the position of the caption in a table.

In science and in STEM areas in general, tables are referenced with a caption. This caption should always be at the bottom for tables and at the top for figures. It is a standard in technical and scientific writing.

Why can't you implement a function that allows to place the caption of a table at the bottom of the table?

**Comment by mcanouil** [comment, marked as answer]
> Why can't you implement a function that allows to place the caption of a table at the bottom of the table?

When asking a question, it's important to use appropriate wording and tone. Please be mindful of how you phrase things.


This being said, how `cap-location`, `fig-cap-location`, and `tbl-cap-location` not enough or does not address the use case described?
Those options are there since a long time now and are documented in several places in the documentation:
- https://quarto.org/docs/reference/formats/html.html#layout

<img width="852" alt="image" src="https://github.com/quarto-dev/quarto-cli/assets/8896044/db83667c-2845-43eb-9cd3-fe55ddd0f228">

**Reply by ajcanepa** [reply]
Hi @mcanouil 
Many thanks for your answer and I'm sorry about the wording. I'm not a native English speaker and the text was AI-Translated. In the original language it didn't sound as bad...sorry fir this!
I'll look for what you sent me.

Many thanks!

**Reply by mcanouil** [reply]
***Off Topic***


>  I'm not a native English speaker and the text was AI-Translated.

I am not a native either (I made and still make some "tone" mistake).
When using AI, you can add the "tone" for example with: "use formal English but keep it friendly", etc.
AI should be used with caution. In the end, you are the one responsible, not the AI.

You can try to translate in your own language the following versions (maybe not using AI, but actual old-school web translator):

- Why can't you implement this? (¿Por qué no puedes implementar esto?)
- Could you implement this? (¿Podrías implementar esto?)
- Would it be possible to implement this? (¿Sería posible implementar esto?)

**Reply by ajcanepa** [reply]
Genial!
Bakán tus consejos, los intento aplicar de ahora en adelante para no venderla más..jeje

**Comment by cscheid** [comment]
> This caption should always be at the bottom for tables and at the top for figures. It is a standard in technical and scientific writing.

FWIW, this is simply not true - I was a practicing academic for 10 years and that's precisely backwards from the journal templates in the venues I published.
